src/components/Button/Button.tsx

```diff
@@ -46,7 +45,7 @@ const ButtonStyled = styled.button<ButtonProps>`
   color: ${(props) => (props.isPrimary ? '#FFFFFF' : '#041d42')};
   background-color: ${(props) => (props.isPrimary ? '#041d42' : '#FFFFFF')};
   padding: 9px 14px 8px;
-  border: ${(props) => (props.isPrimary ? 'none' : '2px solid #041d42')};
+  border: 2px solid #041D42};
```

Comment:
Why did you change the border?

Reply:
Thanks, I didn't even mean to change it.